src/modules/uxrce_dds_client/dds_topics.h.em

```diff
@@ -46,6 +44,7 @@ struct SendSubscription {
 	const char* topic;
 	uint32_t topic_size;
 	UcdrSerializeMethod ucdr_serialize_method;
+	uint16_t max_rate_hz;
```

Comment:
I would either change this to a float (eg I want 0.5 Hz) or convert to an interval in milliseconds earlier.

```suggestion
	float max_rate_hz;
```